Formula/tnftp.rb

```diff
@@ -0,0 +1,38 @@
+class Tnftp < Formula
+  desc "NetBSD's FTP client (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/lukemftp/lukemftp-16.tar.gz"
+  sha256 "ba35a8e3c2e524e5772e729f592ac0978f9027da2433753736e1eb1f1351ae9d"
+
```

Comment:
Probably want a `keg_only :provided_pre_high_sierra` in this formula.

Reply:
@DomT4 Ok